framework/UPGRADE.md

```diff
@@ -65,6 +65,21 @@ Upgrade from Yii 2.0.42
   ```sql
   INSERT INTO `some_table` (`int_unsigned_col`) VALUES (22)
   ```
+* Json::encode() can now handle zero-indexed objects same way as json_encode() and keep them as objects.
+  In order to avoid breaking BC this is enabled by a new option flag.
```

Comment:
```suggestion
* `Json::encode()` can now handle zero-indexed objects in same way as `json_encode()` and keep them as objects. In order to avoid breaking backwards compatibility this behavior could be enabled by a new option flag but is disabled by default.
```